google/cloud/spanner/value.cc

```diff
@@ -111,11 +159,15 @@ std::ostream& StreamHelper(std::ostream& os, google::protobuf::Value const& v,
     case google::spanner::v1::DATE:
       return os << v.string_value();
 
-    case google::spanner::v1::BYTES:
-      return os
-             << "B\""
-             << internal::BytesFromBase64(v.string_value())->get<std::string>()
-             << "\"";
+    case google::spanner::v1::BYTES: {
```

Comment:
Should we do the same escaping for `google::spanner::v1::STRING`?

Reply:
I mentioned this in the PR description. No, I do not think so. I think we should treat STRING more like std::string, which doesn't escape. Bytes is special because it (a) doesn't have an example C++ counterpart, and (b) is more likely to contain non-printable characters.

Reply:
I'm sorry I missed the bit in the PR description.  I still dissent though.  `std::string` also doesn't wrap its output with quotes.  Rendering a `STRING` value is fundamentally different to printing a string of characters.  But, so be it.

Reply:
This PR is about escaping bytes. Let's move the STRING discussion elsewhere. I filed an issue to track it: https://mail.google.com/mail/u/0/#inbox/FMfcgxwHMPfbQplmGvnhKcQcpzvvsnWM